src/tools/compiletest/src/runtest.rs

```diff
@@ -1869,6 +1877,7 @@ impl<'test> TestCx<'test> {
         output_file: TargetLocation,
         emit_metadata: EmitMetadata,
         allow_unused: AllowUnused,
+        link_to_aux: LinkToAux,
```

Comment:
for my sanity: this new argument is necessary because there's no `Command` available until after this function returns it. We might want to refactor at some point to allow passing custom flags without needing all of them to be passed into this function.